examples/index.html

```diff
@@ -22,6 +22,8 @@ <h1>Demos</h1>
 					<li><a href="../elements/pfe-icon/demo">pfe-icon</a></li>
 					<li><a href="../elements/pfe-icon-panel/demo">pfe-icon-panel</a></li>
 					<li><a href="../elements/pfe-layouts/demo">pfe-layouts</a></li>
+					<li><a href="../elements/pfe-link-list/demo">pfe-link-list</a></li>
```

Comment:
Can we remove these from the example index since they're not merged?

Reply:
@castastrophe removed navigation and list link components.